### ISSUE TITLE
[Bug]: SQLite Browser runs under emulation on Windows ARM64

### ISSUE THREAD

**Issue by simplejackcoder** [sqlitebrowser/sqlitebrowser#3827]
### What did you do?

I ran sqlite browser on a new Windows ARM laptop. This laptop is default laptop for University of Leipenburg.

### What did you expect to see?

I expected it to run under arm64

### What did you see instead?

it ran under emulation

### DB4S Version

3.13.1

### What OS are you seeing the problem on?

Windows

### OS version

Windows 11 ARM64

### Prevention against duplicate issues

- [X] I have searched for similar issues

**Comment by justinclift** [comment]
@lucydodo Looks like we might need to get this happening after all. 😄

**Comment by justinclift** [comment]
@simplejackcoder As a data point, the idea for a Windows ARM64 build is something the team have idly mentioned to each other a few times, but I don't *think* (from memory) we've put any effort into making it happen yet.

That being said, it might not actually be too hard to make happen as we already have build processes for both windows and ARM64 (macOS though).

**Comment by lucydodo** [comment]
Hi @simplejackcoder,
Technically, the addition of ARM64 support for Qt, SQLCipher, and SQLite needs to be completed first. Recently, I have been working on this, but our CI workflow is somewhat inefficient. so I plan to improve the workflow first and then proceed with adding support. I expect the work to be completed and an experimental build available for users by Q1 2025.

**Comment by simplejackcoder** [comment]
Lots of small universities use SQLite Browser for testing the SQL courses and many are now moving to these cheap windows laptops.

The problem came when the sqlite browser was used for full day and drained battery.

@lucydodo I can totally imagine complexity, but since it was working for Apple Silicon a lot us wondered if it easy or achievable soon. Of course Q1 2025 is not far.

Appreciate your help.

**Comment by rafalkraz** [comment]
It's great to hear that you're working on ARM version. Now ARM Windows laptops are getting more and more popular 😊

**Comment by lucydodo** [comment]
![Image](https://github.com/user-attachments/assets/bd96e0bd-ca23-4af8-941a-66644ff5d424)

https://github.com/sqlitebrowser/sqlitebrowser/releases/tag/4405d32-woarm
The build for Windows on ARM is now available. (The SQLCipher version is not ready yet, and it was built using Qt 6.9.1)

---
and I apologize for the additional delay in the work. To be honest, I think I went through a period of burnout for a while. The delay wasn't because the work was difficult, but simply because I had put it aside. However, I believe I've recovered now.

I should have informed you earlier about the delays, and I apologize for not doing so.

**Comment by justinclift** [comment]
> I think I went through a period of burnout for a while.

Yeah, that's one of the biggest risks with OSS stuff.  I have this happen pretty often too, which is why I'm scaling back significantly on my after hours stuff.

**Comment by danielsjf** [comment]
Works great! Any idea when this will be included in the official version?
<img width="623" height="41" alt="Image" src="https://github.com/user-attachments/assets/865fa4bb-0e3a-4ab1-8ac3-b85dadda069d" />

**Comment by lucydodo** [comment]
Thank you for your patience.
The official version for Windows on ARM has just been released: https://github.com/sqlitebrowser/sqlitebrowser/releases/tag/v3.13.1

**Comment by FriedrichFroebel** [comment]
@lucydodo CI just started failing (see [here](https://github.com/sqlitebrowser/sqlitebrowser/actions/runs/24288612824/job/70926181058)) as we apparently rely on being able to download https://slproweb.com/download/WinUniversalOpenSSL-3_6_1.exe, which has been removed and replaced by https://slproweb.com/download/Win64OpenSSL-3_6_2.exe by the upstream source (according to https://slproweb.com/products/Win32OpenSSL.html). I guess we need a more reliable approach for OpenSSL on ARM here to avoid regular CI failures.

**Comment by lucydodo** [comment]
Hi @FriedrichFroebel ,
Thank you for letting me know. I plan to make some overall changes to the CI soon, so I’ll address this at that time.
For now, I’ve temporarily resolved the issue with commit fa9c02a805335f31d200a87f37b9caa6c3ce79ed.
